### PR TITLE
Digisam 10 extended metadata

### PR DESCRIPTION
**Pull request by jara2**

### CODE REVIEW

File: src/lang.js [hunk 1]
```diff
@@ -11,8 +11,25 @@ const messages = {
   da
 }
 
+const dateTimeFormats = {
+  'da': {
```
Comment (jorntx) [inline src/lang.js]: No english formats needed? Maybe the are default?

File: src/components/ProductArticle.vue [hunk 2]
```diff
@@ -6,12 +6,26 @@
...
     </figure>
-    <p>
-      Dato: {{ imageResult.response.docs[0].datetime }}
-    </p>
-    <p>
-      keywords: {{ imageResult.response.docs[0].keyword[0] }}
-    </p>
+    <dl>
+      <dt>Dato</dt>
+      <dd>{{ imageResult.response.docs[0].datetime }}</dd>
```
Comment (jorntx) [inline src/components/ProductArticle.vue]: Nothing serious but just and observation - maybe we should make a shorthand for imageResult.response.docs[0]

Comment (jara2) [reply src/components/ProductArticle.vue]: Fixed using computed property.

Comment (jorntx) [reply src/components/ProductArticle.vue]: Nice

File: src/lang.js (hunk 1 above)
Comment (jara2) [reply src/lang.js]: I made english, and yes it was required, but weekday is still written in danish because of OS locale I guess.

Comment (jorntx) [reply src/lang.js]: Odd with the weekday. It should adhere to locale set in app and not OS locale. Otherwise good stuff.

**Pull request merged**